src/main/resources/org/sagebionetworks/web/client/widget/table/v2/results/cell/JSONListCellEditorViewImpl.ui.xml

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+	xmlns:g="urn:import:com.google.gwt.user.client.ui"
+	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
+	xmlns:t="urn:import:org.sagebionetworks.web.client.view.bootstrap.table"
+	xmlns:bg="urn:import:org.gwtbootstrap3.client.ui.gwt"
+	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html">
+
+	<b:FormGroup ui:field="formGroup"
+		addStyleNames="margin-bottom-0-imp">
+		<b:InputGroup width="100%" addStyleNames="show-icon-on-hover">
+			<b:TextBox ui:field="textBox" styleName="form-control min-width-140 cursor-pointer" readOnly="true"/>
```

Comment:
I do wonder if this readOnly style is appropriate (or if it will be confusing to the user, making people think the cell cannot be edited), but we'll see!